### ISSUE TITLE
onRendered helper

### ISSUE THREAD

**Issue by keyscores** [awatson1978/meteor-api#74]
Thanks for this Abigail !

Perhaps the Meteor syntax has changed since this package was originally written. Currently the autocomplete writes:

    Template.name.onRendered = function(){}

When the up-to-date (Sept 2015) syntax should be: 

    Template.name.onRendered( function() { } )

http://docs.meteor.com/#/full/template_onRendered

**Comment by madhan5000** [comment]
I believe the master branch has the new syntax and current tag V2.20.0 that atom editor pulls in is out dated. @awatson1978 Thanks for packaging this ! Could you also create a tag for the latest code in master branch?

**Comment by derwaldgeist** [comment]
+1

I would also like to see this package updated to the latest syntax.

I'm also heaving problems with the implementations of the highlights for special words. For instance, I am using FlowRouter instead of IronRouter, and the package will always only highlight the "Router" part in FlowRouter, which looks quite ugly. Same goes for things like SupportSession.get(), where the Session.get is highlighted. Obviously, the highlighter grammar does not take care of selecting whole words only.

This package needs some love :-)

**Comment by awatson1978** [comment]
Yeah....  so, the folks who develop GitHub and Atom are very clever in general; but whoever implemented the Atom package management kinda screwed things up.  The publication command tries to increment and publish; and gets out of sync with the github tags.  It's super annoying.  Long story short...I've actually had a fix for this for a couple of months, and have tried publishing it a couple of times now, with no success.  Super frustrating.  

I'll give it another look today.  In general, I'm open to accepting pull-requests on any regexes or snippets in this package (provided they don't blatantly deprecate existing functionality; I'd support Router regex updates that include FlowRouter functionality; but not ones that remove IronRouter functionality).  

Given the mechanics of Atom's package manager, however; this package is unlikely to get updated more frequently than a seasonal basis.  

That being said, it would be nice to get the publication functionality working again.

**Comment by cwmp** [comment]
any new activities on this?

**Comment by artiescie** [comment]
One quick and dirty favor - to the incoming stream of meteor users - in case of time shortage -

maybe the atom.io package readme page could add something.  Like an NB: get the latest fixes directly from github repo, after you install the atom package.

**Comment by onedr0p** [comment]
+1 please update
